Readme.md

```diff
@@ -0,0 +1,176 @@
+-   **[Gitflow]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> You can see a Demo here: [Demo](https://claudiarojassoto.github.io/Leaderboard/dist/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To get a local copy up and running, follow these steps.
+> This project requires Node.js and npm installed on your machine.
+
+-Node.js
+-npm
+
+-Clone this repository to your local machine using:
+
+ git clone https://github.com/ClaudiaRojasSoto/Leaderboard.git
+
+-Navigate to the project folder:
+
+ cd Leaderboard
+
+-Install the project dependencies:
+
+npm install
+
+o start the development server, run the following command:
+
+npm start
+
+### Project Structure
+
+The project follows the following folder and file structure:
+
+/src: Contains the source files of the application.
+/src/index.js: Main entry point of the JavaScript application.
+/src/style.css: CSS file for the application styles.
+/src/index.html: Base HTML file of the application.
+/dist: Contains the generated production files.
+webpack.config.js: Webpack configuration file.
+
+### Setup
+
+Clone this repository to your desired folder: https://github.com/ClaudiaRojasSoto/Leaderboard.git
+
+### Install
+
+Install this project with: install WebLint and ESLint
+
+### Usage
+
+To run the project, execute the following command: just need a web Browser
+
+### Run tests
+
+To run tests, run the following command: you just need a simple web browser to run this project for a test
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+-   GitHub: [@githubhandle](https://github.com/ClaudiaRojasSoto)
+-   LinkedIn: [LinkedIn](https://www.linkedin.com/in/claudia-soto-260504208/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+-   [ ] **[Send and receive data from API]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/ClaudiaRojasSoto/Leaderboard/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+> If you like this project show support by following this account
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> -   Microverse for providing the opportunity to learn Git and GitHub in a collaborative environment.
+
+> -   GitHub Docs for providing a wealth of information on Git and GitHub.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT licensed](https://github.com/ClaudiaRojasSoto/Leaderboard/blob/master/MIT.md).
```

Comment:
- [ ] If I want to see your LICENSE, it shows a broken link `404` kindy fix it and your project will be ready to approve.

Reply:
@Rustamxon7 I tried to fix that but I could not, can you give me some light on how to implement this link in a good way, please?

Reply:
I suggest you to copy and past this link:

```
https://github.com/ClaudiaRojasSoto/Leaderboard/blob/basic-page-structure/MIT.md